src/kubernetes/server/server-config.yaml

```diff
@@ -37,8 +37,19 @@ data:
               kubernetes:
                 accounts:
                   default:
+                    imagePullSecret: harborsec
                     limits:
                       memory: 1024Mi
+          container:
```

Comment:
Hi,
Not related to the scope of this PR, but I think that this configuration shouldn't live under 

```
    spring:
      cloud:
        dataflow:
          container:
            metadata:
              disableSslVerification: true
              registry-configurations:
                - registry-host: 'myregistrydomain.com:5000'
```

because we can use the connectivity to a container registry for purposes other than metadata retrieval, for example we could automatically 'import' the images that are under a specific namespace in the registry.

What about   `spring.cloud.dataflow.registries` or `spring.cloud.dataflow.containerRegistries` and under that is the collection that has `registry-host`.  Also, on a per-registry level we should be able to disableSslVerification, so move that down into the list of registries.

Reply:
I agree. I prefer `spring.cloud.dataflow.container.registry` as property prefix. The Registry-Configuration is an inner [List<RegistryConfiguration>](https://github.com/tzolov/spring-cloud-dataflow/blob/improve-k8s-metadata-retrieval/spring-cloud-dataflow-configuration-metadata/src/main/java/org/springframework/cloud/dataflow/configuration/metadata/container/ContainerImageMetadataProperties.java#L58) property. So i don't think we can remove it. 

Also currently this functionality is implemented within the `spring-cloud-dataflow-configuration-metadata` project.  Perhaps it should be moved elsewhere. 

For the record, the following code cries to be replaced with the new registry access: https://github.com/spring-cloud/spring-cloud-dataflow/tree/master/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/validation
Also we should replace the existing `DockerImage` class by the more elaborate `ContainerImage`.